inlong-common/src/main/java/org/apache/inlong/common/enums/DataProxyMsgEncVer.java

```diff
@@ -15,14 +15,19 @@
  * limitations under the License.
  */
 
-package org.apache.inlong.dataproxy.utils;
+package org.apache.inlong.common.enums;
 
-public enum InLongMsgVer {
+/**
+ * Enumeration class of encoding format of data output from DataProxy to MQ
+ */
+public enum DataProxyMsgEncVer {
```

Comment:
It looks more like the type or format of msg encoding than the version.

Reply:
Yes, you're right.